server/metrics/metrics.go

```diff
@@ -75,8 +78,8 @@ var (
 		Name:      "count",
 		Help:      "The total number of invocations whose logs were uploaded to BuildBuddy.",
 	}, []string{
-		// TODO: Slice on build vs. test?
 		InvocationStatusLabel,
```

Comment:
I'm not sure if you want this ordered alphabetically or something. I didn't do this because I was worried about code depending on the order of these fields in this array. I'm pretty sure that can't happen. If you want me to change the ordering here just let me know! Thanks!

Reply:
This looks good -- so far I've been ordering by a rough combination of "most to least interesting" + "ordering consistent with similar metrics"